internal/tiger/cmd/mcp_install.go

```diff
@@ -37,28 +37,44 @@ const (
 	KiroCLI     MCPClient = "kiro-cli"
 )
 
-// TigerMCPServer represents the Tiger MCP server configuration
-type TigerMCPServer struct {
+// MCPServerConfig represents the MCP server configuration
+type MCPServerConfig struct {
 	Command string   `json:"command"`
 	Args    []string `json:"args"`
 }
 
+// InstallOptions configures the MCP server installation behavior
+type InstallOptions struct {
+	// CreateBackup creates a backup of existing config files before modification (default: true for CLI)
+	CreateBackup bool
+	// CustomConfigPath overrides the default config file location
+	CustomConfigPath string
+	// ServerName is the name to register the MCP server as (default: "tiger")
+	ServerName string
+	// Command is the path to the MCP server binary (default: auto-detected)
+	Command string
+	// Args are the arguments to pass to the MCP server binary (default: ["mcp", "start"])
+	Args []string
```

Comment:
I don't think the "defaults" listed in the comments are actually accurate. As far as I can tell, these fields are required, and therefore don't have any "default" values. And in any case, it wouldn't make sense for them to default to Tiger-specific values if this is supposed to be a generic package that can be used to install any MCP server.